contentcuration/contentcuration/frontend/channelList/views/ChannelSet/ChannelSetItem.vue

```diff
@@ -22,61 +22,32 @@
       {{ $formatNumber(channelCount) }}
     </td>
     <td class="text-xs-right">
-      <BaseMenu>
-        <template #activator="{ on }">
-          <VBtn
-            flat
-            block
-            v-on="on"
-          >
-            {{ $tr('options') }}
-            <Icon icon="dropdown" />
-          </VBtn>
-        </template>
-        <VList>
-          <VListTile
-            data-test="edit"
-            :to="channelSetDetailsLink"
-          >
-            <VListTileAction>
-              <Icon icon="edit" />
-            </VListTileAction>
-            <VListTileTitle>{{ $tr('edit') }}</VListTileTitle>
-          </VListTile>
-          <VListTile @click.prevent="deleteDialog = true">
-            <VListTileAction>
-              <Icon icon="trash" />
-            </VListTileAction>
-            <VListTileTitle>{{ $tr('delete') }}</VListTileTitle>
-          </VListTile>
-        </VList>
-      </BaseMenu>
-      <MessageDialog
-        v-model="deleteDialog"
-        :header="$tr('deleteChannelSetTitle')"
-        :text="$tr('deleteChannelSetText')"
+      <KButton
+        :text="$tr('options')"
+        appearance="flat-button"
+        :iconAfter="'dropdown'"
       >
-        <template #buttons="{ close }">
-          <VSpacer />
-          <VBtn
-            flat
-            color="primary"
-            @click="close"
-          >
-            {{ $tr('cancel') }}
-          </VBtn>
-          <VBtn
-            color="primary"
-            data-test="delete"
-            @click="
-              deleteChannelSet(channelSet);
-              close();
-            "
-          >
-            {{ $tr('deleteChannelSetTitle') }}
-          </VBtn>
+        <template #menu>
+          <KDropdownMenu
+            :options="dropdownOptions"
+            :hasIcons="true"
+            position="bottom left"
```

Comment:
Compared to the default, I don't see this resulting in a different experience on any of the resolutions. So I think you can remove `position`.